KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/LiveStreamSettingsModule.java

```diff
@@ -221,11 +226,18 @@ private int getIndex(AMFPacket thisPacket) {
 
 		public void onLivePacket(IMediaStream stream, AMFPacket thisPacket) {
 
+			String streamName = stream.getName();
+
+			if (stream.isTranscodeResult()) {
```

Comment:
plesae test this condition only   first packet (performance)
if (firstPacket && stream.isTranscodeResult()) {